bundles/org.eclipse.equinox.http.servlet/META-INF/MANIFEST.MF

```diff
@@ -8,11 +8,11 @@ Bundle-Activator: org.eclipse.equinox.http.servlet.internal.Activator
 Bundle-Localization: plugin
 Bundle-RequiredExecutionEnvironment: JavaSE-17
 Export-Package: org.eclipse.equinox.http.servlet;version="1.2.0";
-  uses:="org.osgi.framework,
+  uses:="javax.servlet,
+   javax.servlet.http,
    org.eclipse.equinox.http.servlet.internal.servlet,
-   javax.servlet,
+   org.osgi.framework,
```

Comment:
Also here would like to check if internal package is really correct as mentioned by @tjwatson in the other PR

Reply:
I think this is because `HttpServiceServlet` extends (internal) `ProxyServlet` so this is correct!

If someone likes to resolve this `leackage` one would need to make ProxServlet public, but for now we should simply keep it as is.